connector/src/main/java/org/geysermc/connector/utils/ResourcePack.java

```diff
@@ -33,6 +33,10 @@ public static void loadPacks() {
 
         File directory = new File("packs");
 
+	if (!directory.exists()) {
+	    directory.mkdir();
+	}
+	
```

Comment:
Indentation is still a bit off (use spaces instead of tabs).

Reply:
I hope it's right now.